mineral-ui/mineral-ui
# [website]: Use Glamor to inject global css

### Description

* Uses `Glamor` to inject global CSS rather than `css-loader` and `style-loader`
* Improvement of "live" code block legibility on small screens
    * Removes forced line wrapping inside code blocks as this no longer appears to be needed

### Motivation and context

We already use Glamor, so there is no need for those other dependencies and extra configuration.

@kylegach - This removes ~112 packages from node_modules including all of those `postcss`modules that you were inquiring about.

### How to test

1. View contents of styles in header and note that the global styles are injected
1. View "live" code blocks in smaller screens and ensure that they don't break layout, content is horizontally scrollable, lines don't wrap, and that they are generally more legible

### Types of changes

- Other (provide details below)

Refactor of global css loading technique.
Improvement of "live" code block legibility on small screens

### Checklist
